src/components/networks/networkOverviewTab.jsx

```diff
@@ -21,6 +21,7 @@ import PropTypes from 'prop-types';
 import {
     DescriptionList, DescriptionListGroup, DescriptionListTerm, DescriptionListDescription,
     Flex, FlexItem,
+    List, ListItem,
```

Comment:
Maybe this commit, "Utilize PF4 List to display static dhcp mac -> ip mappings" fba4408bc1bd626719f30e9d04df3a1091d23dc1

Should be merged with the previous one? I don't see why it needs to be separate.